src/Xenial.Tasty/Execution/ExecutionContext.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using Xenial.Delicious.Commands;
+using Xenial.Delicious.Metadata;
+using Xenial.Delicious.Scopes;
+
+namespace Xenial.Delicious.Execution
+{
+    public class RuntimeContext : IDisposable
+    {
+        private bool disposedValue;
+
+        public RuntimeContext(TastyScope scope, TestExecutor executor)
```

Comment:
We can omit the scope here, cause executor always needs a scope and we can redirect the scope.